Robinhood/Robinhood.py

```diff
@@ -1,30 +1,44 @@
-"""Robinhood.py: a collection of utilities for working with Robinhood's Private API"""
-import getpass
-import logging
-import warnings
-from enum import Enum
+"""
+    Robinhood.py: a collection of utilities for working with Robinhood's Private API
+"""
 
-import requests
-import six
+from enum import Enum
 from six.moves.urllib.parse import unquote
 from six.moves.urllib.request import getproxies
 from six.moves import input
 from . import exceptions as RH_exception
 
+import getpass
+import requests
+import six
+
+import logging
```

Comment:
PEP import standard:

1. stdlib libraries
2. external dependencies
3. internal imports

Reply:
Nice catch - just saw this [here](https://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#Imports_formatting) and fixed it.

I always thought it was external first so that the user/developer knew to install those first, followed by application-specific since that is next-most relevant, followed by stdlib since those were included by default.